src/styles/blocks/card/blockscard/body.scss

```diff
@@ -0,0 +1,10 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: normal;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
The `body` class does not seem to be related to the `card` block, according to the BEM methodology. It's better to rename this class to something like `card__body` if it's an element of the `card` block. If it's a separate block, it should have its own file.